### ISSUE TITLE
[Enhancement] RadioButton needs a Value property

### ISSUE THREAD

**Issue by tjdennis** [xamarin/Xamarin.Forms#11516]
## Summary

A RadioButton needs more than just a Text property. The Text could be in any language if localization is used and would not match a value stored in your database. A Value property provides a consistent value that could be stored and retrieved from the database.  Every other UI framework has a Value property available. (HTML, WPF, WinForms, etc).

The Value should be bindable so that you can point all radio buttons in a group to a MVVM property and have the grouping select which one to show as selected.

This is still an experimental control so it should be a safe time to add it.

## API Changes
Add a Value property to the RadioButton class.
Grouping should be done on the Value, not the Text.

```
<RadioButton GroupName="MyGroup" Text="Oui" Value="YES"/>
<RadioButton GroupName="MyGroup" Text="No" Value="NO"/>

```
See Plugin.InputKit.Shared.Controls.RadioButton in the Xamarin.Forms.InputKit project as an example of one with the Value property.

**Comment by levelfivecoder** [comment]
Anyone started development?

**Comment by hartez** [comment]
Included as part of #11628
